bucket/sublime-text.json

```diff
@@ -8,27 +8,61 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://download.sublimetext.com/Sublime%20Text%20Build%203211%20x64.zip",
-            "hash": "fb77731fa3eda907620a9e8b85bdccc5b5cb4dcf655dcfdc6c7591b78abd0f31"
+            "url": [
+                "https://download.sublimetext.com/Sublime%20Text%20Build%203211%20x64.zip",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/sublime-text/sublime-text-install-context.reg",
```

Comment:
Since there is nested folder file is not needed to be prefixed with `sublime-text-` or is there a special reason to keep it?

Reply:
I think there is no need tho keep it now